spec/oil-slice-range.test.sh

```diff
@@ -89,21 +89,9 @@ write @x
 5
```

Comment:
Good question... It's definitely kinda counter-intuitive. Even in python, I didn't know this was a thing until very recently. If we expand the `:`-syntax for slices to include an optional step, this will become redundant, though